openmetadata-ui/src/main/resources/ui/src/components/Auth/AppAuthenticators/GenericAuthenticator.tsx

```diff
@@ -43,6 +47,7 @@ export const GenericAuthenticator = forwardRef(
 
     const handleSilentSignIn = async () => {
       const resp = await renewToken();
+      setOidcToken(resp.accessToken);
```

Comment:
exception handling

Reply:
It's in the parent where its being handled